cloudos_cli/__main__.py

```diff
@@ -410,9 +410,26 @@ def run(apikey,
     else:
         docker_login = False
     if nextflow_version == 'latest':
-        nextflow_version = '24.04.4'
-        print('[Message] You have specified Nextflow version \'latest\'. The workflow will use the ' +
+        if execution_platform == 'aws':
+            nextflow_version = '24.04.4'
+        elif execution_platform == 'azure':
+            nextflow_version = '22.11.1-edge'
+        else:
+            nextflow_version = '22.10.8'
+        print(f'[Message] You have specified Nextflow version \'latest\' for execution platform \'{execution_platform}\'. The workflow will use the ' +
               f'latest version available on CloudOS: {nextflow_version}.')
+    if execution_platform == 'aws':
+        if nextflow_version not in ['24.04.4', '22.10.8']:
```

Comment:
```suggestion
        if nextflow_version not in AWS_NEXTFLOW_VERSIONS:
```

Could we add three new global vars at the beginning of the script:
```
AWS_NEXTFLOW_VERSIONS = ['22.10.8', '24.04.4']
AZURE_NEXTFLOW_VERSIONS = ['22.11.1-edge']
HPC_NEXTFLOW_VERSIONS = ['22.10.8']
```
Doing so we can easily update. For the same reason, it would also be nice to have another set of global vars for the latest version of each execution_platform:
```
AWS_NEXTFLOW_LATEST = '24.04.4'
AZURE_NEXTFLOW_LATEST = '22.11.1-edge4'
HPC_NEXTFLOW_LATEST = '22.10.8'
```